right click on artificial token without morphology produces interpolation error

add an artificial token 
right click on the token to bring up the context menu
error reported in javascript console:
arethusa_packages.min.js:96 Error: [$interpolate:interr] http://errors.angularjs.org/1.2.17/$interpolate/interr?p0=%7B%7B%20plugin.c…p1=TypeError%3A%20Cannot%20read%20property%20'attributes'%20of%20undefined
    at Error (native)
    at http://www.perseids.org/tools/arethusa/dist/arethusa_packages.min.js:11:450
    at Object.r (http://www.perseids.org/tools/arethusa/dist/arethusa_packages.min.js:84:446)
    at h.$digest (http://www.perseids.org/tools/arethusa/dist/arethusa_packages.min.js:113:321)
    at h.$apply (http://www.perseids.org/tools/arethusa/dist/arethusa_packages.min.js:116:451)
    at e (http://www.perseids.org/tools/arethusa/dist/arethusa.min.js:2:3761)
    at HTMLSpanElement.<anonymous> (http://www.perseids.org/tools/arethusa/dist/arethusa.min.js:2:4056)
    at HTMLSpanElement.n.event.special.(anonymous function).handle (http://www.perseids.org/tools/arethusa/dist/arethusa_packages.min.js:3:10147)
    at HTMLSpanElement.n.event.dispatch (http://www.perseids.org/tools/arethusa/dist/arethusa_packages.min.js:3:6444)
    at HTMLSpanElement.r.handle (http://www.perseids.org/tools/arethusa/dist/arethusa_packages.min.js:3:3219)

--- a/app/js/arethusa.morph/morph.js
+++ b/app/js/arethusa.morph/morph.js
@@ -654,6 +654,7 @@ angular.module('arethusa.morph').service('morph', [
       var id = token.id;
       var forms = new Forms(token.string);
       self.analyses[id] = forms;
+      token.morphology = {};
       loadToken(forms, id);
     });
 

